crm_claim_rma/models/account_invoice_line.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
```

Comment:
You don't need to split this submodel in another file. The rule only applies to main models, but not by this kind of submodels that are always attached to its parent.